Fix flake8 issues

There has been an oversight with some issues coming from SH2 that flake8 has detected, a fix needs to be made in order to apply the next release:

app/modules/dataset/tests/test_selenium.py:182:121: E501 line too long (135 > 120 characters)
app/modules/dataset/tests/test_unit.py:3:1: F401 'shutil' imported but unused
app/modules/dataset/tests/test_unit.py:4:1: F401 'flask.url_for' imported but unused
app/modules/dataset/tests/test_unit.py:5:1: F401 'flask.request' imported but unused
app/modules/dataset/tests/test_unit.py:7:1: F401 'datetime.datetime' imported but unused
app/modules/dataset/tests/test_unit.py:15:1: F811 redefinition of unused 'db' from line 6

--- a/app/modules/dataset/tests/test_selenium.py
+++ b/app/modules/dataset/tests/test_selenium.py
@@ -178,8 +178,9 @@ constraints
     Video | Audio => "Media Player\""""
 
             # Verify content matches expected
+            test = "Test failed: File content does not match expected content."
             assert file_content_text.strip() == expected_content.strip(), \
-                f'Test failed: File content does not match expected content.\nExpected:\n{expected_content}\nGot:\n{file_content_text}'
+                f'{test}\nExpected:\n{expected_content}\nGot:\n{file_content_text}'
         except NoSuchElementException:
             raise AssertionError('Test failed: File content is not visible.')
 

--- a/app/modules/dataset/tests/test_unit.py
+++ b/app/modules/dataset/tests/test_unit.py
@@ -1,10 +1,5 @@
 import os
 import pytest
-import shutil
-from flask import url_for
-from flask import request
-from app import db
-from datetime import datetime
 from app.modules.auth.models import User
 from app.modules.dataset.models import (
     DataSet, DSMetaData, DatasetStatus, PublicationType

